gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -415,10 +426,17 @@ bool ROOT::Experimental::TWebWindowsManager::Show(ROOT::Experimental::TWebWindow
       TestProg(prog, "/usr/bin/chrome-browser");
 #endif
       is_chrome = prog.Length()>0;
+#ifdef _MSC_VER
+      if (win.IsBatchMode())
+         exec = gEnv->GetValue("WebGui.ChromeBatch", "fork: --headless --disable-gpu --remote-debugging-port=8090 --enable-logging $url");
```

Comment:
>   --remote-debugging-port=8090 

This is not very elegant solution. One other should try same workaround, like in Linux 

    --remote-debugging-socket-fd=0

or at least number should be configurable via gEnv

Reply:
OK, will test again with `--remote-debugging-socket-fd=0`